data/database.py

```diff
@@ -1840,6 +1840,47 @@ class RepoMirrorConfig(BaseModel):
     root_rule = ForeignKeyField(RepoMirrorRule)
 
 
+@unique
+class IndexStatus(IntEnum):
+    """
+    Possible statuses of manifest security scan progress.
+    """
+
+    FAILED = -1
+    UNSUPPORTED = 0
+    TIMED_OUT = 1
+    WAITING = 2
+    IN_PROGRESS = 3
+    COMPLETED = 4
+
+
+@unique
+class IndexerVersion(IntEnum):
+    """
+    Possible versions of security indexers.
+    """
+
+    V2 = 0
```

Comment:
Why would we have `V2` if Clair v2 implementation does not consume `ManifestSecurityStatus` data model?

Reply:
Because we're going to be changing it to consume the MSS. That's how we can get off of the Image table without fully migrating to Clair V4